docs/user-guide/mvd-configuration.md

```diff
@@ -848,17 +915,17 @@ The API returns the following information in a JSON response:
 | API                                                       | Description                                                  |
 | --------------------------------------------------------- | ------------------------------------------------------------ |
 | /server (GET)                                             | Returns a list of accessible server endpoints for the Zowe Application Server. |
-| /server/config (GET)                                      | Returns the Zowe Application Server configuration from the `zluxserver.json` file. |
+| /server/config (GET)                                      | Returns the Zowe Application Server configuration from the `server.json` file. |
 | /server/log (GET)                                         | Returns the contents of the Zowe Application Server log file. |
 | /server/loglevels (GET)                                   | Returns the verbosity levels set in the Zowe Application Server logger. |
 | /server/environment (GET)                                 | Returns Zowe Application Server environment information, such as the operating system version, node server version, and process ID. |
 | /server/reload (GET)                                      | Reloads the Zowe Application Server. Only available in cluster mode. |
 | /server/agent (GET)                                       | Returns a list of accessible server endpoints for the ZSS server. |
-| /server/agent/config (GET)                                | Returns the ZSS server configuration from the `zluxserver.json` file. |
+| /server/agent/config (GET)                                | Returns the ZSS server configuration from the `server.json` file. |
```

Comment:
Doesn't this return a merged version of the config from server.json and zowe.yaml?